src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
@pavarnos I don't know how you are planning to do the same for the trait. If I can see some code it may help. Yes we could probably ask someone on twitter ;) .